src/pal/prebuilt/inc/clrallevents.h

```diff
@@ -0,0 +1,2218 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Do not modify the file. This file is generated using the script from <root>/src/inc/genXplatEtw.pl
+//
+
+extern "C" BOOL  EventEnabledGCStart();
+extern "C" ULONG   FireEtwGCStart(const unsigned int  Count,
```

Comment:
I see.  The two spaces for pointer/non-pointer types is not an issue, but the spaces before arguments makes things much easier to read.